src/components/authorization/policies/by-role/consultant.policy.ts

```diff
@@ -22,6 +22,14 @@ import { inherit, member, Policy, Role } from '../util';
   r.Partnership.when(member).read,
   r.PeriodicReport.read,
   r.Product.read,
+  r.ProgressReport.when(member).read,
+  r.ProgressReportVarianceExplanation.when(member).read,
+  r.ProgressReportWorkflowEvent.when(member).read,
+  [
+    r.ProgressReportCommunityStory,
+    r.ProgressReportHighlight,
+    r.ProgressReportTeamNews,
+  ].flatMap((it) => [it.read.specifically((p) => [p.responses.read])]),
```

Comment:
I think the idea is to allow CM's to see these as well for low/medium sens projects that that are _NOT_ a member of, right? If so, we need to call this out in CM too with that _or_ condition.